spec/helpers/locations_helper_spec.rb

```diff
@@ -51,14 +51,10 @@
     let(:location) { FactoryBot.create(:location, startdate: nil) }
 
     before do
-      Timecop.freeze(2023, 10, 10)
+      allow(Time).to receive(:now).and_return(Time.zone.local(2023, 10, 10))
```

Comment:
I'm sure you tried to remove the stubbing of now entirely, but I'm slightly surprised that we're stubbing time but there's nothing in the context or describe blocks above it about being in a given collection year